pkg/controllers/queue/queue_controller.go

```diff
@@ -96,22 +137,64 @@ func NewQueueController(
 		DeleteFunc: c.deletePodGroup,
 	})
 
+	c.cmdInformer = informerfactory.NewSharedInformerFactory(c.vcClient, 0).Bus().V1alpha1().Commands()
+	c.cmdInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: func(obj interface{}) bool {
+			switch obj.(type) {
+			case *busv1alpha1.Command:
+				cmd := obj.(*busv1alpha1.Command)
+				if cmd.TargetObject != nil &&
+					cmd.TargetObject.APIVersion == schedulingv1alpha2.SchemeGroupVersion.String() &&
```

Comment:
let's create `QueueKind` in apis package and use DeepEqual here.

Reply:
Since the `TargetObject` is an instantiated  object that contains the `name` and `uid` of queue reference, it is difficult to use the `DeepEqual` function to compare it with a constant with `QueueKind` type. A filter function is provided to replace the filtering action here.